lua/rzls/server/methods/hover.lua

```diff
@@ -59,9 +53,10 @@ return function(err, result, ctx, config)
     }, nil, razor_bufnr)
 
     if response and response.result ~= nil and response.result.ranges[1] ~= nil then
-        vim.lsp.handlers.hover(err, {
-            contents = hover_result.result.contents,
+        ---@type lsp.Hover
+        return {
+            contents = hover_result.result.contents.value,
```

Comment:
Should we just forward the content from the virtual LSP as is? Not sure if it was matters tho, I tried both ways and they seem to yield the same results.
```suggestion
            contents = hover_result.result.contents,
```